python/datasources/data_source.py

```diff
@@ -19,19 +19,25 @@ def get_table_name():
     def get_attr(self, attributes, key):
         attr = attributes.get(key)
         if attr is None:
-            raise RuntimeError("Attribute: {} not found on payload".format(key))
+            raise RuntimeError(
+                "Attribute: {} not found on payload".format(key))
         return attr
 
     def upload_to_gcs(self, gcs_bucket, **attrs):
-        """Attempts to download a file from a url and upload as a
+        """
+        Attempts to download a file from a url and upload as a
```

Comment:
What about the other data sources, don't they need to be updated (or is that going to be a followup?)

Reply:
We can add those to this change